doc/source/parameters/cfd/laser_heat_source.rst

```diff
@@ -8,10 +8,11 @@ If a laser heat source is present in a simulation, it can be added in this secti
 
   subsection laser parameters
     set enable               = false
+    set type                 = exponential decay
     set concentration factor = 2.0
     set power                = 100.0
     set absorptivity         = 0.5
-    set penetration depth    = 0.0
+    set penetration depth    = 1.0
     set beam radius          = 0.0
     set start time           = 0.0
     set end time             = 1.0
```

Comment:
so you changed the default value?

Reply:
It was changed because it became incoherent in the sense that it does a division by 0 (by default) when the laser is activated. And the radiation application-test (``heat_transfert_vof_radiation``) was failing after I added my assertion on the penetration depth value. When talking with @hepap, she said we should be able to use the radiative cooling with or without the laser, and since the power is by default 0.0 changing the depth default value should't hinder much. However, I now realize that the default power value is not 0.0, that's my mistake. I could change it... but I'm now starting to wonder if the radiative cooling should belong in the ``laser`` subsection, since we should be able to use it without the laser also, even though that wouldn't be it's primary use.

Reply:
Yeah i guess teh radiative cooling would not really just belong to laser although right now it's the only context I can think of us using it to be honest...